sample/src/main/res/layout/fragment_bar.xml

```diff
@@ -6,17 +6,22 @@
     android:layout_height="match_parent"
     tools:context=".ui.bar.BarFragment">
 
-    <TextView
-        android:id="@+id/text_dashboard"
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/chartBar"
```

Comment:
pls use snake case convention for xml id naming

Reply:
Let me handle this one... Will fix it in HORIZONTAL BarChart PR.